apps/quickstart-integrations-monday-code/src/queue-service.js

```diff
@@ -0,0 +1,33 @@
+import { Logger } from "@mondaycom/apps-sdk";
+import { PubSub } from "@google-cloud/pubsub";
+const pubSubClient = new PubSub();
+
+const logTag = "QueueService";
+const logger = new Logger(logTag);
+
+
+export const produceMessage = async ({ body }) => {
+    const topicName = process.env.MNDY_TOPIC_NAME
```

Comment:
where does this env variable come from - is it part of the GCP environment or does it need to be added by the user?

Reply:
yes, we're injecting it, user just needs to consume it

Reply:
@ShayElkana maybe we can move it to the SDK as the developer in any case have 0 control over it?